cephfs/cephfs_test.go

```diff
@@ -122,15 +188,18 @@ func TestReleaseMount(t *testing.T) {
 	assert.NoError(t, err)
 	require.NotNil(t, mount)
 
-	err = mount.Release()
-	assert.NoError(t, err)
+	assert.NoError(t, mount.Release())
+	assert.NoError(t, mount.Release())
```

Comment:
is it intended to run this twice?

Reply:
Yes. I'll add a comment to clarify why it is being called a 2nd time.